src/Process/FeatureProcessing.js

```diff
@@ -68,7 +68,19 @@ export default {
                 if (layer.onMeshCreated) {
                     layer.onMeshCreated(result);
                 }
-
+                result.traverse((o) => {
+                    if (o.material) {
```

Comment:
Actually, what about putting this update code above, for example somewhere after line 36 (by looping on `features`). After all, the update function is made for this kind of things:
- all these properties would automatically get updated at each frame
- we won't need to bother about parentLayer in View.js (basically, we won't need any change to View.js in this PR any more)
- we won't need to set layer.object3d to parentLayer.object3d in WFS_Provider.

The reasons why I think it's better:
- assigning `parentLayer.object3d` to `layer.object3d` breaks the assumption that a layer's object3d is its *own* object, with which it can do whatever it wants. I'd like to keep that assumption for layers that do have an object3d. So no object3d actually gives the correct information in this case: wfs layers cannot act on a single object3d to reach all its node.
- itowns already provides a mechanism that allows you to traverse each node of a layer: it's precisely the layer `update` function. This mechanism is already battle-tested, and made to be efficient (we don't do it more than 1 time per frame for example), and you actually launch it by calling `notifyChange` anyway. Here, if you attach the opacity to a slider, it would guarantee that we won't traverse the object tree more than one time per frame (click and input events can trigger a lot faster).

Sorry, I should have started with this, but I didn't realize it at first.